app/views/partials/_primary_nav.html.erb

```diff
@@ -76,7 +76,7 @@
 
       <% if current_user.logged_in? %>
         <span class="navbar-text">
-          Hello, <%= current_user.display_name %>
+          Hello, <%= link_to current_user.display_name, current_user %>
```

Comment:
Could the name of the current user be highlighted or underlined? It would be clear to me as a user that this is a link.